config/database.yml

```diff
@@ -20,10 +21,8 @@ test:
 
 development:
   <<: *default
-  user: <%= ENV.fetch('PG_USER') %>
-  password: <%= ENV.fetch('PG_PASSWORD') %>
+  database: signup-back-development
```

Comment:
In my opinion, this could break this installation instruction : https://github.com/betagouv/datapass/blame/40b8e44ee6b1b2b45cff17e8e460cc0e44ebd75c/README.md#L102

What do you think ?

Reply:
maybe the right behavior should be fetch the PG_DATABASE with default value set to `signup-back-development`

Reply:
Good point indeed, I'm going to apply your suggestion

Reply:
I just applied your suggestion (and rebased because of Gemfile.lock conflict)